util/deephash/deephash_test.go

```diff
@@ -324,6 +324,55 @@ func BenchmarkHash(b *testing.B) {
 	}
 }
 
+func ptrTo[T any](v T) *T { return &v }
+
+// filterRules is a 1-element FilterRule with everything populated.
```

Comment:
1-element?

Reply:
Updated comment to more match the commit message.